src/cargo.rs

```diff
@@ -106,12 +106,8 @@ impl Crate {
         }
     }
 
-    /// Create a `config.toml` in `.cargo` with a `[target]` and `[unstable]` section.
```

Comment:
I'm not sure this (pub) method is for now used by anything else but the `pio` builder, but - with that said - why change the signature?

I would suggest to leave it as-is and then just pass `None` for the target from the `pio::project` call-site?

Reply:
ahh sorry, i didn’t see it was part of the public api. done in dea89a1bed5b9dbbe62b6da0867b70c7ab4f5a1f :)